docs/pkg-set.8

```diff
@@ -94,12 +94,12 @@ Also see
 .Xr pkg-updating 8
 and
 .Sx EXAMPLES .
-.It Fl x , Cm --regex
+.It Fl x , Fl -regex
 Match
 .Ar pkg-name
-as a regular expression according to the "modern" or "extended" syntax of
+as a regular expression using the modern or extended syntax of
 .Xr re_format 7 .
-.It Fl v Ar 01
+.It Fl v Ar 0|1 , Fl -vital Ar 0|1
```

Comment:
In style.mdoc, we do say that that the preferred syntax for "x | y" is without a space, however the vast majority of the time we do use a space, and so does the rest of the industry. The other day I even found this syntax with a space describing sauces available with chicken wings.